index.html

```diff
@@ -1,23 +1,66 @@
 <!DOCTYPE html>
 <html>
-	<body>
-		<h1>My First Heading</h1>
-		<p>My first paragraph.</p>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
+</head>
 
-		<div id="target"></div>
+<body>
+<div class="container">
+<div class="row">
```

Comment:
Use whitespace to indent your html tags.  Some editors will do this for you.  What are you using?

Reply:
Will do. I am using Visual studio code.